package.json

```diff
@@ -13,17 +13,26 @@
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
     "@fontsource/roboto": "^5.0.13",
+    "@fortawesome/fontawesome-free": "^6.5.2",
     "@mui/icons-material": "^5.15.18",
     "@mui/material": "^5.15.18",
     "@mui/styled-engine-sc": "^6.0.0-alpha.18",
+    "@types/axios": "^0.14.0",
     "axios": "^1.7.2",
-    "react": "^18.2.0",
+    "formik": "^2.4.6",
```

Comment:
We have both Formik and react-hook-form installed.

Reply:
Everyone used Formik, and I had a problem with it, so I used hook form until I found a solution to the problem, and the problem was solved, and the code will be changed.